src/EscapeHit.Invoice/Queries/GetInvoiceById/GetInvoiceByIdResultMapping.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace EscapeHit.Invoice.Queries.GetInvoiceById
+{
+
+    public interface IGetInvoiceByIdResultMapping
+    {
+        GetInvoiceByIdResult MapFrom(InvoiceEntity entity);
+    }
+
+    public class GetInvoiceByIdResultMapping : IGetInvoiceByIdResultMapping
+    {
+        public GetInvoiceByIdResult MapFrom(InvoiceEntity entity)
+        {
+            if (entity is null)
```

Comment:
@viktor-kulkov  I don't like this null return, but without it, 404 is not handled well when the entity is not found in  `OkOrNotFound()` extension method. Pls advice.